### ISSUE TITLE
Add custom fontFamily

### ISSUE THREAD

**Issue by abolinhas** [xotahal/react-native-material-ui#301]
Hi,
I can't add a custom fontFamily
I tried.
const uiTheme = {
    fontFamily: 'Questrial',
    palette: {
        primaryColor: COLOR.green900,
    },
    toolbar: {
        container: {
            height: 50,
            backgroundColor: '#005447',
        },
        titleText: {
            fontFamily: "Questrial",
        }
    },
};

But the font is not load, still with Roboto.
But if I use
<Text style={{fontFamily: "Questrial"}}>HI</Text>
The text will use the Questrial font
Best regards

**Comment by peterchibunna** [comment]
I have experienced this too.

**Comment by marktkimball** [comment]
I'm having the same issue too.  I was trying to add the style to the `centerElement` of the `<Toolbar />` component (which is just a text) and couldn't get it to work.

**Comment by peterchibunna** [comment]
Please @xotahal  how can we go about this? I want to help if you can guide me. I looked into some of the components, and there's no reference to font family; and I wish I can tackle this.

**Comment by onooma** [comment]
Same issue here.

**Comment by xotahal** [comment]
That's weird guys - according to [this](https://github.com/xotahal/react-native-material-ui/blob/master/src/Toolbar/CenterElement.react.js#L149), the `titleText` style should have been applied. Need to debug what's happening ...

**Comment by juliusspencer** [comment]
I have the same issue with a button:

`<MaterialButton style={{ text: { color: '#66BBB0', fontSize: 32, fontFamily: 'Montserrat-Medium', } }} text="1"/>`

**Comment by Majid110** [comment]
I have the same issue

**Comment by moiseshilario** [comment]
The same thing happens in the <Button> component. Which seems to be implemented correctly [here](https://github.com/xotahal/react-native-material-ui/blob/master/src/Button/Button.react.js#L210). Other style properties like `color, fontSize` works normally. Wonder if the component doesn't recognize the font or the font gets overridden somewhere else

**Comment by moiseshilario** [comment]
Ok, so I discovered what was happening. In `/react-native-material-ui/src/styles/typography.js` he is setting a specific `fontWeight` for each component (see [here](https://github.com/xotahal/react-native-material-ui/blob/master/src/styles/typography.js#L19)). So you have to override it on your styles to the right `fontWeight` of your custom font. In my case, I was trying to change the  font for the Button component, which was using [`fontWeight: medium (500)`](https://github.com/xotahal/react-native-material-ui/blob/master/src/styles/typography.js#L4) by default, and my custom font required `'400'`, so I just changed it, and the font worked like a charm 

```js
<Button
      style={{ container: styles.button, text: styles.buttonText }}
      primary
      raised
      text={item.title}
      key={item.title}
      upperCase={false}
      onPress={() => this.navigation.navigate('Chapters', { book: item })}
    />
...

STYLES

 buttonText: {
    fontSize: 16,
    fontFamily: 'RobotoSlab-Regular',
    fontWeight: '400',
  },
```


;D

TLDR:
Just add the right `fontWeight`  for your custom fontFamily as a string (e.g `fontWeight: '400'`)
